src/index.js

```diff
@@ -61,14 +58,13 @@ function organizeImports(unsortedCode, extension) {
 const parsers = {
   typescript: {
     ...typescriptParsers.typescript,
-    preprocess(text) {
-      return organizeImports(text, '.ts')
+    preprocess(text, opts) {
+      return organizeImports(text, '.ts', opts.filepath)
     }
-  },
```

Comment:
Removing this bracket+comma is breaking the syntax of this file. I assume that was unintentional, but this PR is clearly broken as-is.

Reply:
Yeahhh, nice catch. I'm going to fix it.